lib/telegram2discord/setup.js

```diff
@@ -278,6 +278,23 @@ function setup(tgBot, dcBot, dcUsers, messageMap) {
 			// Find out who the message is from
 			let fromName = getDisplayName(tgMessage.from, tgMessage.chat);
 
+			// Check if it is a reply
```

Comment:
This is very wet code. It is (near?) identical to some code further up. Maybe extract this part into a function which is then called when needed?